c/include/libsbp/sbp.h

```diff
@@ -76,12 +81,20 @@ typedef struct {
   u8 msg_buff[256];
   void* io_context;
   sbp_msg_callbacks_node_t* sbp_msg_callbacks_head;
+  sbp_allmsg_fn_t allmsg_fn;
```

Comment:
No a huge fall of `allmsg`, would prefer `all_msg`, but I can live with it.

Reply:
I just didn't want anyone to be confused that all_msg meant all messages.

Reply:
agree on this point, considering one of the use cases of this function (hence the different signature) is to receive the entire raw frame so the binary blob can be written to file. from this perspective I would almost prefer a name like `sbp_raw_frame_handler` or something to that effect